library/crates/Cargo.toml

```diff
@@ -52,7 +54,9 @@ serde = "=1.0.152"
 serde_json = "=1.0"
 serial_test = "=0.9.0"
 sha2 = "=0.10.6"
+smol = "=1.3.0"
 tokio = { version = "=1.24.1", features = ["rt", "rt-multi-thread"] }
+tokio-stream = "=0.1.11"
```

Comment:
Do we know why Tokio packages their stream library as a separate crate, as opposed to what they normally do, namely shipping it as a **feature** of `tokio` crate itself?

Reply:
TL;DR: tokio's minimum supported Rust version does not support streams.

https://docs.rs/tokio/latest/tokio/stream/

> Originally, we had planned to ship Tokio 1.0 with a stable Stream type but unfortunately the [RFC](https://github.com/rust-lang/rfcs/pull/2996) had not been merged in time for Stream to reach std on a stable compiler in time for the 1.0 release of Tokio. <...> [O]nce `Stream` has made it into the standard library and the MSRV period has passed, we will implement stream for our different types.